core/shared/src/main/scala/monocle/Traversal.scala

```diff
@@ -225,6 +226,13 @@ object PTraversal extends TraversalInstances {
         Applicative[F].apply6(f(get1(s)), f(get2(s)), f(get3(s)), f(get4(s)), f(get5(s)), f(get6(s)))(_set(_, _, _, _, _, _, s))
     }
 
+  def apply[S, T, A, B](f: S => FreeAp[IndexedStore[A, B, ?], T]) =
```

Comment:
I think that using the same name for both would be ok. Eventually, if users adopted `Traversal`'s constructor as default, it would be nice to provide an `apply` alias for it.